pages/tidal tools/vsphere.md

```diff
@@ -6,20 +6,58 @@ sidebar: main_sidebar
 permalink: vsphere.html
 folder: tidaltools
 ---
-## Getting Started
-To get started you will need to make sure you have a copy of [Tidal Tools](https://get.tidal.sh) and it is [installed](tidal-tools.html).
+
+## Overview
+
+As part of your Cloud Migration journey, it is very important to have a reliable inventory of your infrastructure. Tidal Tools is able to connect to a Hypervisor to synchronize your inventory of virtual machines from vSphere.
+
+This guide will walk you through the steps and necessary configuration to properly integrate with vSphere and capture the following data points on its VMs.
+
+```
+- FQDN
+- IP Addresses
+- HostName
+- AssignedID (InstanceUuid)
+- RAM Used (gb)
+- RAMAllocated (gb)
+- CPU Name
+- CPU Count
+- Storage Used (gb)
+- Storage Allocated (gb)
+- Operating System
+- Operating System Version
+- CreatedAt
+- UpdatedAt
+```
```

Comment:
How about we remove the code block from here to make it aligned with the other lists of fields? Like what we have on importing page [link](https://guides.tidalmg.com/import-from-excel.html#importing-servers:~:text=when%20importing%20Servers.-,Default%20fields%3A,-Host%20Name%20(*)).

```suggestion
- FQDN
- IP Addresses
- HostName
- AssignedID (InstanceUuid)
- RAM Used (gb)
- RAMAllocated (gb)
- CPU Name
- CPU Count
- Storage Used (gb)
- Storage Allocated (gb)
- Operating System
- Operating System Version
- CreatedAt
- UpdatedAt
```

Reply:
Good idea, Thanks @pranavmalaviya 👍